package hu.bme.hit.exceptions


class AuthError(msg: String) : Exception(msg) {
}